docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -231,6 +240,82 @@ Each test run has its own test context.
 
 > `t.ctx` is initialized with an empty object without a prototype. You can iterate its keys without the `hasOwnProperty` check.
 
+### Fixture Hooks
+
+Fixture hooks are executed before the first test in a fixture is started and after the last test is finished.
```

Comment:
I guess it's worth mentioning that unlike test hooks these hooks executed completely on server side and only once per fixture. Also it would be nice to provide possible use case, e.g. setup or teardown tested page server or something like that.